tools/NuGet/template-artifactory/DynamoVisualProgramming.DynamoCoreRuntimeNET60.nuspec

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+    <metadata>
+        <id>DynamoVisualProgramming.DynamoCoreRuntimeNET60</id>
+        <version>$Version$</version>
+        <authors>Autodesk</authors>
+        <owners>Autodesk</owners>
+        <license type="expression">Apache-2.0</license>
+        <projectUrl>https://github.com/DynamoDS/Dynamo</projectUrl>
+        <icon>content\images\logo.png</icon>
+        <requireLicenseAcceptance>false</requireLicenseAcceptance>
+        <description>Assemblies required to start a DynamoModel and execute DesignScript code bundled along with their dependencies.
+            Built targeting Windows Net6 with exclusion of test libraries and symbol files. </description>
+        <copyright>Copyright Autodesk 2023</copyright>
+        <dependencies>
+            <group targetFramework="net6.0">
+            </group>
+        </dependencies>
+    </metadata>
+    <!--for now we assume that that the working directory root is dynamo/bin/AnyCPU/Release-->
```

Comment:
This should be `Dynamo\bin\NET60_Windows\Release`

Reply:
No @aparajit-pratap See below, when you define the files, you are actually navigating from the working dir